src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/directionissued/DirectionIssuedAboutToSubmitHandler.java

```diff
@@ -255,8 +256,7 @@ private PreSubmitCallbackResponse<SscsCaseData> validateForPdfAndCreateCallbackR
                 new PreSubmitCallbackResponse<>(caseData);
 
         DocumentLink url = null;
-
-        if (nonNull(caseData.getDocumentStaging().getPreviewDocument()) && callback.getEvent() == EventType.DIRECTION_ISSUED) {
+        if (isYes(callback.getCaseDetails().getCaseData().getDocumentGeneration().getGenerateNotice()) && nonNull(caseData.getDocumentStaging().getPreviewDocument()) && callback.getEvent() == EventType.DIRECTION_ISSUED) {
```

Comment:
checkstyle is probably going to complain about this, as it usually asks for '&&' to be at the start of the line
indentation seems to be off as well